[Fix] 예외 핸들러 컴파일 에러

## 🐞 어떤 버그인가요?
> 어떤 버그인지 간결하게 설명해주세요
- CI/CD 진행 중 컴파일 에러로 인한 build 중단
- `GlobalExceptionHandler`에서 `Event `관련 예외 클래스들을 찾을 수 없음
## 🤔 어떤 상황에서 발생한 버그인가요?
> (가능하면) Given-When-Then 형식으로 서술해주세요
- given : #28 `GlobalExceptionHandler`에 `Event `관련 예외 핸들러 구현
- when : CI/CD 파이프라인 실행
- then : `EventCreationFailedException`, `EventUpdateFailedException`, `EventDeletionFailedException `클래스를 찾을 수 없다는 컴파일 에러 발생

## 📋 예상 결과
> 정상적으로 작동했을 때 기대되는 결과를 작성해주세요
- Event 관련 예외 클래스가 정상적으로 컴파일 완료
- CI/CD 파이프라인이 정상적으로 진행
## 🔍 실제 결과
> 실제로 발생한 문제 상황을 작성해주세요
```text
Run gradle/gradle-build-action@v2
Gradle User Home already exists: will not restore from cache.
/home/runner/work/10th-ODIT-BE/10th-ODIT-BE/gradlew clean build -x test
Downloading https://services.gradle.org/distributions/gradle-8.11.1-bin.zip
.............10%.............[20](https://github.com/IT-Cotato/10th-ODIT-BE/actions/runs/12813537252/job/35728068247#step:6:21)%.............30%.............40%.............50%.............60%.............70%.............80%.............90%.............100%

Welcome to Gradle 8.11.1!

Here are the highlights of this release:
 - Parallel load and store for Configuration Cache
 - Java compilation errors at the end of the build output
 - Consolidated report for warnings and deprecations

For more details see https://docs.gradle.org/8.11.1/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)
> Task :clean

> Task :compileJava FAILED
/home/runner/work/10th-ODIT-BE/10th-ODIT-BE/src/main/java/com/adit/backend/global/error/GlobalExceptionHandler.java:200: error: cannot find symbol
	protected ResponseEntity<ErrorResponse> handleEventCreationFailedException(EventCreationFailedException ex) {
	                                                                           ^
  symbol:   class EventCreationFailedException
  location: class GlobalExceptionHandler
/home/runner/work/10th-ODIT-BE/10th-ODIT-BE/src/main/java/com/adit/backend/global/error/GlobalExceptionHandler.java:[21](https://github.com/IT-Cotato/10th-ODIT-BE/actions/runs/12813537252/job/35728068247#step:6:22)3: error: cannot find symbol
	protected ResponseEntity<ErrorResponse> handleEventUpdateFailedException(EventUpdateFailedException ex) {
	                                                                         ^
  symbol:   class EventUpdateFailedException
  location: class GlobalExceptionHandler
/home/runner/work/10th-ODIT-BE/10th-ODIT-BE/src/main/java/com/adit/backend/global/error/GlobalExceptionHandler.java:[22](https://github.com/IT-Cotato/10th-ODIT-BE/actions/runs/12813537252/job/35728068247#step:6:23)6: error: cannot find symbol
	protected ResponseEntity<ErrorResponse> handleEventDeletionFailedException(EventDeletionFailedException ex) {
	                                                                           ^
  symbol:   class EventDeletionFailedException
  location: class GlobalExceptionHandler
/home/runner/work/10th-ODIT-BE/10th-ODIT-BE/src/main/java/com/adit/backend/global/error/GlobalExceptionHandler.java:199: error: cannot find symbol
	@ExceptionHandler(EventCreationFailedException.class)
	                  ^
  symbol:   class EventCreationFailedException
  location: class GlobalExceptionHandler
/home/runner/work/10th-ODIT-BE/10th-ODIT-BE/src/main/java/com/adit/backend/global/error/GlobalExceptionHandler.java:212: error: cannot find symbol
	@ExceptionHandler(EventUpdateFailedException.class)
	                  ^
  symbol:   class EventUpdateFailedException
  location: class GlobalExceptionHandler
/home/runner/work/10th-ODIT-BE/10th-ODIT-BE/src/main/java/com/adit/backend/global/error/GlobalExceptionHandler.java:2[25](https://github.com/IT-Cotato/10th-ODIT-BE/actions/runs/12813537252/job/35728068247#step:6:26): error: cannot find symbol
	@ExceptionHandler(EventDeletionFailedException.class)
	                  ^
  symbol:   class EventDeletionFailedException
  location: class GlobalExceptionHandler
6 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Compilation failed; see the compiler output below.
  /home/runner/work/10th-ODIT-BE/10th-ODIT-BE/src/main/java/com/adit/backend/global/error/GlobalExceptionHandler.java:200: error: cannot find symbol
  	protected ResponseEntity<ErrorResponse> handleEventCreationFailedException(EventCreationFailedException ex) {
  	                                                                           ^
    symbol:   class EventCreationFailedException
    location: class GlobalExceptionHandler
  /home/runner/work/10th-ODIT-BE/10th-ODIT-BE/src/main/java/com/adit/backend/global/error/GlobalExceptionHandler.java:213: error: cannot find symbol
[Incubating] Problems report is available at: file:///home/runner/work/10th-ODIT-BE/10th-ODIT-BE/build/reports/problems/problems-report.html
  	protected ResponseEntity<ErrorResponse> handleEventUpdateFailedException(EventUpdateFailedException ex) {
  	                                                                         ^
    symbol:   class EventUpdateFailedException
    location: class GlobalExceptionHandler
  /home/runner/work/10th-ODIT-BE/10th-ODIT-BE/src/main/java/com/adit/backend/global/error/GlobalExceptionHandler.java:2[26](https://github.com/IT-Cotato/10th-ODIT-BE/actions/runs/12813537252/job/35728068247#step:6:27): error: cannot find symbol
  	protected ResponseEntity<ErrorResponse> handleEventDeletionFailedException(EventDeletionFailedException ex) {
  	                                                                           ^
    symbol:   class EventDeletionFailedException
    location: class GlobalExceptionHandler
  /home/runner/work/10th-ODIT-BE/10th-ODIT-BE/src/main/java/com/adit/backend/global/error/GlobalExceptionHandler.java:199: error: cannot find symbol
  	@ExceptionHandler(EventCreationFailedException.class)
  	                  ^
    symbol:   class EventCreationFailedException
    location: class GlobalExceptionHandler
  /home/runner/work/10th-ODIT-BE/10th-ODIT-BE/src/main/java/com/adit/backend/global/error/GlobalExceptionHandler.java:212: error: cannot find symbol
  	@ExceptionHandler(EventUpdateFailedException.class)
  	                  ^
    symbol:   class EventUpdateFailedException
    location: class GlobalExceptionHandler
  /home/runner/work/10th-ODIT-BE/10th-ODIT-BE/src/main/java/com/adit/backend/global/error/GlobalExceptionHandler.java:225: error: cannot find symbol
  	@ExceptionHandler(EventDeletionFailedException.class)
  	                  ^
    symbol:   class EventDeletionFailedException
    location: class GlobalExceptionHandler
  6 errors

* Try:
> Check your code and dependencies to fix the compilation error(s)
> Run with --scan to get full insights.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.11.1/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
2 actionable tasks: 2 executed

BUILD FAILED in 20s
Error: Gradle build failed: see console output for details
```

## 📎 추가 정보
- 스크린샷이나 에러 로그가 있다면 첨부해주세요

--- a/src/main/java/com/adit/backend/domain/event/service/query/EventQueryService.java
+++ b/src/main/java/com/adit/backend/domain/event/service/query/EventQueryService.java
@@ -29,7 +29,7 @@ public class EventQueryService {
 
     public EventResponseDto getEventById(Long id) {
         Event event = eventRepository.findById(id)
-                .orElseThrow(() -> new EntityNotFoundException("Event not found"));
+                .orElseThrow(() -> new EntityNotFoundException("이벤트를 찾을 수 없습니다."));
         return eventConverter.toResponse(event);
     }
 

--- a/src/main/java/com/adit/backend/global/error/GlobalExceptionHandler.java
+++ b/src/main/java/com/adit/backend/global/error/GlobalExceptionHandler.java
@@ -196,12 +196,12 @@ public class GlobalExceptionHandler {
 	 * @param ex EventCreationFailedException
 	 * @return ResponseEntity<ErrorResponse>
 	 */
-	@ExceptionHandler(EventCreationFailedException.class)
-	protected ResponseEntity<ErrorResponse> handleEventCreationFailedException(EventCreationFailedException ex) {
-		log.error("EventCreationFailedException", ex);
-		final ErrorResponse response = ErrorResponse.of(GlobalErrorCode.EVENT_CREATION_FAILED, ex.getMessage());
-		return new ResponseEntity<>(response, HttpStatus.OK);
-	}
+	// @ExceptionHandler(EventCreationFailedException.class)
+	// protected ResponseEntity<ErrorResponse> handleEventCreationFailedException(EventCreationFailedException ex) {
+	// 	log.error("EventCreationFailedException", ex);
+	// 	final ErrorResponse response = ErrorResponse.of(GlobalErrorCode.EVENT_CREATION_FAILED, ex.getMessage());
+	// 	return new ResponseEntity<>(response, HttpStatus.OK);
+	// }
 
 	/**
 	 * [Exception] 이벤트 업데이트 실패 시 발생한 경우
@@ -209,12 +209,12 @@ public class GlobalExceptionHandler {
 	 * @param ex EventUpdateFailedException
 	 * @return ResponseEntity<ErrorResponse>
 	 */
-	@ExceptionHandler(EventUpdateFailedException.class)
-	protected ResponseEntity<ErrorResponse> handleEventUpdateFailedException(EventUpdateFailedException ex) {
-		log.error("EventUpdateFailedException", ex);
-		final ErrorResponse response = ErrorResponse.of(GlobalErrorCode.EVENT_UPDATE_FAILED, ex.getMessage());
-		return new ResponseEntity<>(response, HttpStatus.OK);
-	}
+	// @ExceptionHandler(EventUpdateFailedException.class)
+	// protected ResponseEntity<ErrorResponse> handleEventUpdateFailedException(EventUpdateFailedException ex) {
+	// 	log.error("EventUpdateFailedException", ex);
+	// 	final ErrorResponse response = ErrorResponse.of(GlobalErrorCode.EVENT_UPDATE_FAILED, ex.getMessage());
+	// 	return new ResponseEntity<>(response, HttpStatus.OK);
+	// }
 
 	/**
 	 * [Exception] 이벤트 삭제 실패 시 발생한 경우
@@ -222,11 +222,11 @@ public class GlobalExceptionHandler {
 	 * @param ex EventDeletionFailedException
 	 * @return ResponseEntity<ErrorResponse>
 	 */
-	@ExceptionHandler(EventDeletionFailedException.class)
-	protected ResponseEntity<ErrorResponse> handleEventDeletionFailedException(EventDeletionFailedException ex) {
-		log.error("EventDeletionFailedException", ex);
-		final ErrorResponse response = ErrorResponse.of(GlobalErrorCode.EVENT_DELETION_FAILED, ex.getMessage());
-		return new ResponseEntity<>(response, HttpStatus.OK);
-	}
+	// @ExceptionHandler(EventDeletionFailedException.class)
+	// protected ResponseEntity<ErrorResponse> handleEventDeletionFailedException(EventDeletionFailedException ex) {
+	// 	log.error("EventDeletionFailedException", ex);
+	// 	final ErrorResponse response = ErrorResponse.of(GlobalErrorCode.EVENT_DELETION_FAILED, ex.getMessage());
+	// 	return new ResponseEntity<>(response, HttpStatus.OK);
+	// }
 
 }

